modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/edit_entry.jsp

```diff
@@ -170,7 +170,7 @@ renderResponse.setTitle((entry != null) ? BlogsEntryUtil.getDisplayTitle(resourc
 								contents="<%= content %>"
 								editorName='<%= PropsUtil.get("editor.wysiwyg.portal-web.docroot.html.portlet.blogs.edit_entry.jsp") %>'
 								name="contentEditor"
-								onChangeMethod="onChangeEditor"
+								onChangeMethod="onChangeContentEditor"
```

Comment:
You could also avoid passing an `onChange` method and manually bind to it using `window['contentEditor'].getNativeEditor()` which should give you access to the `AlloyEditor` or `CKEditor` instance.

The main issue is that the editor here is configurable as per `<%= PropsUtil.get("editor.wysiwyg.portal-web.docroot.html.portlet.blogs.edit_entry.jsp") %>`, so if you want to keep that, this is the only way to pass a normalized callback from Java to JS 🤷